addons/mail/static/tests/discuss_app/jump_to_present.test.js

```diff
@@ -234,3 +235,59 @@ test("Post message when seeing old message should jump to present", async () =>
         after: [".o-mail-Message-content", { text: "Most Recent!" }], // should load around present
     });
 });
+
+test("jump to present triggers with at least 10 long messages seen", async () => {
```

Comment:
We should add a positive assertion to this test.

Reply:
Last step had to assert presence of the banner but I forgot it. I made the change :)

Reply:
Also, I don't think the "seen" keyword accurately reflects what is going on here. The test is complicated, and I'd like the test name to better convey its meaning.